GUI/SearchTaskWindow.xaml.cs

```diff
@@ -394,7 +402,32 @@ private void ApmdExpander_Collapsed(object sender, RoutedEventArgs e)
             dataContextForSearchTaskWindow.SearchModeExpanderTitle = "Some search properties...";
         }
 
+        private static void PreviewIfInt(object sender, TextCompositionEventArgs e)
```

Comment:
Some values may be decimals, some may not be negative. I believe different textboxs should have slightly varying rules. Would you go over those?